src/main/resources/reference.conf

```diff
@@ -167,6 +168,7 @@ resourceTypes = {
           controlled-user-shared-workspace-resource = ["editor", "writer", "reader"]
           controlled-application-shared-workspace-resource = ["writer", "reader"]
           google-project = ["pet-creator"]
+          wds-instance = ["writer"]
```

Comment:
This made me think about naming. I'm trying to decide if it makes more sense to have something specific & traceable like "wds-instance", or something generic with an angle towards supporting multiple dataservices (for which we have no plans). I lean towards wds-instance, since we avoid creating new names & more confusion.

I'm also thinking about control plane vs dataplane permissions. Seeing permissions for a wds-instance makes me think: "Does this give permission to create and delete WDS instances?" Right now we definitely don't have permissions at that granularity, so this is a bit of a future question.

Reply:
Agreed on future permissions. I could imagine a lot of granularity here, such as permissions to change schema vs. write to the existing schema, permission to delete data vs. insert/update, permission to clone, etc. etc.

For now I'm trying to keep it as simple as possible, just the permissions I can see us needing for MVP. The "delete" permission may be necessary if/when the parent workspace is deleted and we need to delete (from Sam) the child `wds-instance` resources.